client/components/domains/use-my-domain/utilities/get-availability-error-message.js

```diff
@@ -19,6 +24,18 @@ export function getAvailabilityErrorMessage( { availabilityData, domainName, sel
 			);
 		}
 
+		if ( errorAction ) {
```

Comment:
After looking at the code way more closely, this naming doesn't feel right, because we're not actually taking action in response to an error.

I think we should be much more specific so it's more apparent what the prop is for. Maybe we can go for something a bit more self-descriptive like `registerNowAction` or `notRegisteredAction`?

```suggestion
		if ( registerNowAction || notRegisteredAction ) {
```